server/api.go

```diff
@@ -142,29 +143,29 @@ func (p *Plugin) isJaaSMeeting(path string) bool {
 
 func (p *Plugin) handleJaaSBundle(w http.ResponseWriter, r *http.Request) {
 	if !p.getConfiguration().UseJaaS {
-		http.Error(w, "Not found", http.StatusFound)
+		http.Error(w, "jaas not enabled", http.StatusFound)
```

Comment:
```suggestion
		http.Error(w, "Not found", http.StatusNotFound)
```
I think the error was right before and status code should be "not found".